packages/framework/src/Form/Admin/Product/Brand/BrandFormType.php

```diff
@@ -80,22 +82,68 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ];
         }
 
-        $builder
+        $builderBasicInformationGroup = $builder->create('basicInformation', GroupType::class, [
+            'label' => t('Basic information'),
+        ]);
+        $builderBasicInformationGroup
+            ->add('id', DisplayOnlyType::class, [
```

Comment:
The `id` shouldn't be a part of the form at all when a new Brand is created (i.e. when `$options['brand']` is `null`).

See the new Brand page on *master*.